compiler-rt/test/hwasan/TestCases/enable-disable.c

```diff
@@ -0,0 +1,39 @@
+// RUN: %clang_hwasan -O1 %s -o %t && %run %t 2>&1
+
```

Comment:
Maybe add a comment on the purpose of this test (I know we don't usually, but would be good to start).

Reply:
Done